src/executors/kernel_constructor.hpp

```diff
@@ -86,6 +86,7 @@ Specialised case for using_local_memory == subgroup, which contains a subgroup
 local accessor.
 @tparam value_t Value type of the accessor.
 */
+#if 0
```

Comment:
Marking this so it doesn't get forgotten

Reply:
Fixed